tools/atip/atip/web/steps.py

```diff
@@ -179,3 +181,101 @@ def i_accept_alert(context):
 @step(u'I should see an alert with text "{text}"')
 def should_see_alert_text(context, text):
     assert context.app.get_alert_text() == text, u'Text was not found'
+
+
+@step(u'I wait {n:d} seconds')
+def wait_senconds(context,n):
+   time.sleep(n)
+
+
+@step(u'I go to frame "{key}"')
+def i_visit_frame(context, key):
+    context.app.driver.switch_to_frame(key)
+    assert True
+
+
+@step(u'I go out of frame')
+def i_jump_frame(context):
+    context.app.driver.switch_to_default_content()
+    assert True
+
+
+@step(u'I should see nothing in "{attr}" attr of "{key}" area')
+def check_text_by_key(context, attr, key):
+    elements = context.app.driver.find_elements_by_id(key)
+    if len(elements) == 1 and elements[0].get_attribute(attr) == "":
+        assert True
+    else:
+        assert False
+
+
+@step(u'I should see "{text}" with "{color}" color in "{key}" area')
+def should_see_text_element_with_color(context, text, key, color):
+    assert context.app.check_normal_text_element_timeout_with_color(
+        text, key, color), u'Text was not found'
+
+
+@step(u'I should see "{key}" area in "{color}" color')
+def should_see_text_element_with_color(context, key, color):
+    assert context.app.check_normal_element_timeout_with_color(
+        key, color), u'The background color is wrong'
+
+
+@step(u'I verify value in "{key}" is "{expecttype}" type')
+def check_type_by_key(context, key, expecttype):
+    typename = context.app.check_content_type(key, display=True)
+    if typename == expecttype:
+        assert True
+    else:
+        assert False
+
+
+@step(u'I save div "{key}" as "{pic_name}"')
+def save_div(context, key, pic_name):
+    assert context.app.save_div_as_picture(key, pic_name, width=0, height=0)
+
+
+@step(u'I save div "{key}" as "{pic_name}" with width "{width}" and height "{height}"')
+def save_div(context, key, pic_name, width, height):
+    assert context.app.save_div_as_picture(key, pic_name, width, height)
+
+
+@step(u'pic "{pic1}" and pic "{pic2}" should be more than "{similarity}" similar')
```

Comment:
why not put the steps into common folder? as the called function under common.
Same requirement for other steps whose function under common.